include/robot_fingers/trifinger_platform.hpp

```diff
@@ -0,0 +1,110 @@
+/**
+ * @file
+ * @brief TODO
+ * @copyright 2020, Max Planck Gesellschaft.  All rights reserved.
+ * @license BSD 3-clause
+ */
+#pragma once
+
+// TODO move to a separate package to not add unnecessary dependencies to
+// robot_fingers?
+#include <robot_interfaces/finger_types.hpp>
+#include <trifinger_object_tracking/object_tracker_frontend.hpp>
+
+namespace robot_fingers
+{
+// TODO move implementation to cpp file
+// NOTE: Inheriting from Frontend makes implementation easier but means that we
+// cannot rename get_observation
+class TriFingerPlatform : public robot_interfaces::TriFingerTypes::Frontend
```

Comment:
ich wuerde wahrscheinlich TriFingerPlatformFrontend nennen :). ist etwas lang, aber faende ich consistenter

Reply:
i would probably not derive from the Trifinger frontend, but just have a pointer to it, as you do for the tracker.

i am aware that this would lead to a bit more redundant code, but i would find it more logical, and you would also have more control over which functions are available in this class (e.g. i don't know if it makes sense to have a get_observation function in this class, or if we do, it should probably include both, the one from the robot and the one from the tracker) so if deriving at all, i would probably derive from RobotFrontend<Action, Observation> where the Observation includes both..